scene/gui/tree.cpp

```diff
@@ -6147,6 +6171,11 @@ Tree::Tree() {
 	range_click_timer->connect("timeout", callable_mp(this, &Tree::_range_click_timeout));
 	add_child(range_click_timer, false, INTERNAL_MODE_FRONT);
 
+	dropping_unfold_timer = memnew(Timer);
+	dropping_unfold_timer->set_wait_time(0.6);
```

Comment:
This duration should be exposed as a project setting, so you can adjust it for accessibility reasons. Check the implementation of the `"gui/timers/incremental_search_max_interval_msec"` project setting across the codebase for a reference.

Also, the default duration feels a bit long. It might be worth trying to reduce it to 0.5 seconds, but I don't know if this will have adverse effects on a trackpad or touch input (Android editor). I'll let others confirm this first though.

Reply:
Oh thanks for bringing that up, I meant to ask about it. I used https://github.com/godotengine/godot/pull/41138 as a reference for this, and I figured it'd be good for both unfolding behaviors to share a timing configuration. I just wasn't sure where to put it. I'll add that and connect it to the tree and the inspector :+1:

I saw some code that lead me to believe that drag-n-drop is disabled for touch interactions, but I may have misread.

Reply:
Actually... I'm thinking this should be an editor setting, not a project setting, and add a parameter to the tree to control it.

If users of the Tree node want trees to have different hover times for some reason, it would allow that. It would also allow different collaborators of the same godot project to have their own settings, which is more accessible.